.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="7.0" LINT_CHECK="1"
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install geojson Shapely
+  - sudo apt-get -q -y install postgis
+
+script:
+  - travis_run_flake8
```

Comment:
Calling `travis_run_flake8` is deprecated now that it's done in `travis_run_tests`